crates/env/Cargo.toml

```diff
@@ -48,6 +48,8 @@ secp256k1 = { version = "0.27.0", features = ["recovery", "global-context"], opt
 #
 # Sadly couldn't be marked as dev-dependency.
 # Never use this crate outside the off-chain environment!
+scale-decode = { version = "0.5.0", default-features = false, optional = true }
+scale-encode = { version = "0.1.0", default-features = false, optional = true }
```

Comment:
Same here, don't need to import these, just use `subxt::ext` reexports

Reply:
but we don't have here any access to `subxt`, do we?